Cargo.toml

```diff
@@ -20,9 +20,12 @@ serde = { version = "1.0.136", features = ["derive"] }
 serde_json = "1.0.79"
 node-bindgen = "5.1.0"
 flate2 = "1.0.24"
-fluvio = { version = "0.18.1", features = ["admin"] }
+fluvio = { version = "0.18.2", features = ["admin"] }
```

Comment:
I think that we should move from using crates.io to github tags here, like how we do in connectors